modules/lang-painless/src/test/java/org/elasticsearch/painless/BaseClassTests.java

```diff
@@ -131,7 +131,7 @@ public void testNoArgs() throws Exception {
                 scriptEngine.compile("testNoArgs3", "_score", NoArgs.CONTEXT, emptyMap()));
         assertEquals("cannot resolve symbol [_score]", e.getMessage());
 
-        String debug = Debugger.toString(NoArgs.class, "int i = 0", new CompilerSettings());
+        String debug = Debugger.toString(NoArgs.class, "int i = 0", new CompilerSettings(), Whitelist.BASE_WHITELISTS);
```

Comment:
This method can take a `List<Whitelist>` now so I can test the output using the fancy binding changes. To make sure the constants really get inlined.